### DISCUSSION TITLE
Error while running jenkins frontend Job

### DISCUSSION THREAD

**Discussion by bharath-gaveni** [learndevopsonline/learndevopsonline#1612] [Q&A] [answered]
Error while running jenkins frontend Job

![Screenshot (8)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/e71ab441-ce0f-431a-862a-315bb11bf2d5)

![Screenshot (9)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/d0447435-42fc-4312-99c9-cbd6a4fd1829)

**Comment by sreedharm07** [comment]
@bharath-gaveni ping your workstation ip

**Comment by bharath-gaveni** [comment]
3.85.117.240 - public 
172.31.42.123 - private

**Reply by sreedharm07** [reply, marked as answer]
Hi @bharath-gaveni 
from your workstation not able to ssh your frontend,sql,backend instances. there is no route to ssh. thats why the error ( not able to ssh ) happening
am able to check your instances but not your aws infra so please check your infra

check -
route table associations
security groups
peering connections between workstation vpc and another vpc

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/867d62f7-0566-48da-97f5-2b5d95a52a2a)

**Comment by bharath-gaveni** [comment]
okay thank you @sreedharm07 now its working fine.
